base/floatfuncs.jl

```diff
@@ -214,6 +214,17 @@ approximately equal component-wise.
 The binary operator `≈` is equivalent to `isapprox` with the default arguments, and `x ≉ y`
 is equivalent to `!isapprox(x,y)`.
 
+Note that comparing `x ≈ 0` (i.e., comparing to zero with the default tolerances)
```

Comment:
"comparing" -> "writing" (you don't compare `x ≈ 0` to anything).

BTW, I'm not sure it's common to say "you", "your" in the docs. I'd say imperative is more common, and "the problem (at hand)" is fine.

Reply:
Neither "comparing" nor "writing" is correct.  "writing `x ≈ 0`" is not equivalent to "`x == 0`", it is not even equivalent to "writing `x == 0`".  Only `x ≈ 0` is equivalent to "`x == 0`".